Entities/ACrab.cpp

```diff
@@ -2357,37 +2377,28 @@ void ACrab::Update()
     ////////////////////////////////////
     // Aiming
 
-	//float rotAngle = GetRotAngle();
+    // Get rotation angle of crab
+    float RotAngle = GetRotAngle();
 
-	//if (IsHFlipped())
-	//	rotAngle = -rotAngle;
+    // Adjust AimRange limits to crab rotation
+    float m_AdjustedAimRangeUpperLimit = (m_HFlipped) ? m_AimRangeUpperLimit - RotAngle : m_AimRangeUpperLimit + RotAngle;
```

Comment:
These upper and lower limits here shouldn't start with m_ cause m_ is only for member variables (i.e. ones that are defined in header and maintain their value throughout the object's existence). These should just be called adjustedAimRangeUpperLimit and adjustedAimRangeLowerLimit.